clang/lib/CodeGen/Targets/AArch64.cpp

```diff
@@ -814,6 +820,49 @@ Address AArch64ABIInfo::EmitMSVAArg(CodeGenFunction &CGF, Address VAListAddr,
                           /*allowHigherAlign*/ false);
 }
 
+class SMEAttributes {
```

Comment:
I'd like to avoid adding another `SMEAttributes class`. Could you write some specific functions instead? i.e.

```
static bool isStreaming(const FunctionDecl *F) { ... }
static bool isStreamingCompatible(const FunctionDecl *F) { ... }
```

Reply:
Done.